pkg/kube/client.go

```diff
@@ -47,8 +47,13 @@ import (
 	cmdutil "k8s.io/kubectl/pkg/cmd/util"
 )
 
-// ErrNoObjectsVisited indicates that during a visit operation, no matching objects were found.
-var ErrNoObjectsVisited = errors.New("no objects visited")
+var (
+	// ErrNoObjectsVisited indicates that during a visit operation, no matching objects were found.
+	ErrNoObjectsVisited = errors.New("no objects visited")
+
+	// ErrClusterUnreachable indicates that the Kubernetes cluster cannot be reached
+	ErrClusterUnreachable = errors.New("Kubernetes cluster unreachable")
```

Comment:
If we go ahead and abort the upgrade on all errors (except the releaseNotFound error), then technically I don't need the  modifications in this file.  But I find it is nice to have, in case it is useful in the future.  But it does introduce a new API.  

Keep or remove the new `ErrClusterUnreachable` constant?

Reply:
@bacongobbler do you want to keep this part?  It is not technically needed.

Reply:
This error message is no longer a constant on the latest master.  I'll simply remove the fact that I create a variable for it.